### PR TITLE
feat(driver): add some iphone viewport

### PR DESCRIPTION
**Pull request by ryota-murakami**
### User facing changelog
* Added viewport preset of `iphone-7`, `iphone-8`, `iphone-se2`

#### sake: improve Developer Experience
viewport preset consider useful most people because people tend to memorize mobile device naming e.g. `google-pixel`, `samsung galaxy` etc clearly more than device pixels number.

So, This time I added 3 devices that has exact same resolution as a `iphone-6`(750 x 1334)
source: https://devhints.io/resolutions source hosted github repo over 9,000 and maintain frequency. Seems trusted. 

#### this is draft pull request
In the first place I don't know Cypress dev team has positive feeling about support more viewport,

so I started super low regression risk part.

If Cypress dev team had agree with positive, I'll add some popular devices viewport e.g. `google-pixel` series, `galaxy s20`, `ipad-pro`, `ipad-air`, series, `ipad-3`, `ipad-4`, `ipad-5`, `ipad-6`, `ipad-7`, `ipad-8`(modern days, call that `ipad-8gen` generation style also popular on net news, SNS etc.

### How has the user experience changed?
Cypress user can use familiar mobile device screen size easily, less research time about cypress viewport preset detail.

Unlike me, I have `iphone-7` in real life so I attempt use it viewport, but I can't figure out next number of `iphone-7`... :thinking::question::sob:

### PR Tasks
* [x] Have tests been added/updated?
~~- [ ] Has the original issue been tagged with a release in ZenHub? ~~
* [x] Has a PR for user-facing changes been opened in https://docs.cypress.io/api/commands/viewport.html#Arguments ? 
* [x] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/compare/develop...ryota-murakami:add-iphone-viewport?expand=1#diff-2b7d393932ee4e054d8e4371309dba98)?
~~- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?~~

Every feedback welcome, I'm glad got comments some thought when you have plenty time and your smartphone doesn't exist in viewport. :sweat_smile:

Thank you for reading about this! :hugs:

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
@ryota-murakami I see this is still in Draft, is this ready for review or will you have time to finish this PR?

**Comment by ryota-murakami** [comment]
@jennifer-shehane Thank you mentioning it, It's ready for review so I just update PR status from Draft!

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
Can you update the types to reflect the new presets? https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts#L5298:L5298

Also, could you open a pull request against our [docs](https://github.com/cypress-io/cypress-documentation) to document the new viewport presets?

File: cli/types/cypress.d.ts [hunk 1]
```diff
@@ -5295,7 +5295,7 @@ declare namespace Cypress {
 
   type Encodings = 'ascii' | 'base64' | 'binary' | 'hex' | 'latin1' | 'utf8' | 'utf-8' | 'ucs2' | 'ucs-2' | 'utf16le' | 'utf-16le'
   type PositionType = 'topLeft' | 'top' | 'topRight' | 'left' | 'center' | 'right' | 'bottomLeft' | 'bottom' | 'bottomRight'
-  type ViewportPreset = 'macbook-15' | 'macbook-13' | 'macbook-11' | 'ipad-2' | 'ipad-mini' | 'iphone-xr' | 'iphone-x' | 'iphone-6+' | 'iphone-6' | 'iphone-5' | 'iphone-4' | 'iphone-3' | 'samsung-s10' | 'samsung-note9'
+  type ViewportPreset = 'macbook-15' | 'macbook-13' | 'macbook-11' | 'ipad-2' | 'ipad-mini' | 'iphone-xr' | 'iphone-x' | 'iphone-6+' | 'iphone-se2' | 'iphone-8' | 'iphone-7' | 'iphone-6' | 'iphone-5' | 'iphone-4' | 'iphone-3' | 'samsung-s10' | 'samsung-note9'
```
Comment (ryota-murakami) [inline cli/types/cypress.d.ts:5298]: @jennifer-shehane 

> Can you update the types to reflect the new presets? https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts#L5298:L5298

I think that work including this PR, isn't it? 🤔

**Comment by ryota-murakami** [comment]
@jennifer-shehane 

> Also, could you open a pull request against our docs to document the new viewport presets?

Sure, I'll update docs viewport presets section and raise PR! 😀

File: cli/types/cypress.d.ts (hunk 1 above)
Comment (jennifer-shehane) [reply cli/types/cypress.d.ts:5298]: Sorry, you're right 🙃

**Comment by jennifer-shehane** [comment]
@ryota-murakami I think for future new viewport sizes, we're thinking to pull in some outside dependency where we can request viewports / sizes and use those as the available viewports. Because keeping up with an arbitrary list that people add to just might get out of hand. Just so that answers your question about newer viewports. Maybe there's a library that lists those we can pull in.

That or maybe there's a way to reference or pull in Google Chrome device sizes that they use in emulation mode. Ideally our viewports would match those. 

Screen Shot 2020-10-09 at 2 26 39 PM

File: cli/types/cypress.d.ts (hunk 1 above)
Comment (ryota-murakami) [reply cli/types/cypress.d.ts:5298]: @jennifer-shehane Never mind! I appreciate your review! 😀

**Comment by ryota-murakami** [comment]
> Also, could you open a pull request against our docs to document the new viewport presets?

@jennifer-shehane I send PR to Docs repo that added exact same iphone medels.
https://github.com/cypress-io/cypress-documentation/pull/3229

**Comment by ryota-murakami** [comment]
@jennifer-shehane Thank you for your answer about viewport direction.

> Maybe there's a library that lists those we can pull in.

That's might be cool, I'm looking forward to that! 🤗

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**